BookClub.Tests/Models/BookTest.cs

```diff
@@ -25,7 +26,86 @@ public void BookHasNecessaryProperties()
                     ISBN = "020161622X",
                     CoverUrl = "https://images-na.ssl-images-amazon.com/images/I/41BKx1AxQWL._SX258_BO1,204,203,200_.jpg"
                 };
+                var ValidationResult = new List<ValidationResult>();
+                var actual = Validator.TryValidateObject(book, new ValidationContext(book), ValidationResult, true);
+                Assert.True(actual);
             }
+
+            [Fact]
+            public void BookHasAnISBNWithMoreThanThirteenDigits()
```

Comment:
`BookHasAnIBNWithMoreThanThirtheenDigits_ReturnsValidationError()` would be a more descriptive name